source/features/useful-not-found-page.tsx

```diff
@@ -71,29 +71,37 @@ async function getUrlToFileOnDefaultBranch(): Promise<string | void> {
 }
 
 async function showMissingPart(): Promise<void> {
-	const parts = parseCurrentURL();
-	const bar = <h2 className="container mt-4 text-center"/>;
+	const pathParts = parseCurrentURL();
+	const breadcrumbs: HTMLElement[] = [];
+
+	for (const [i, part] of pathParts.entries()) {
+		// Exclude parts that don't exist as standalones
+		if (i === 0 && part === 'orgs') {
+			continue;
+		}
 
-	for (const [i, part] of parts.entries()) {
 		if (i === 2 && ['tree', 'blob', 'edit'].includes(part)) {
-			// Exclude parts that don't exist as standalones
 			continue;
 		}
 
-		if (i === parts.length - 1) {
+		if (i === pathParts.length - 1) {
 			// The last part of the URL is a known 404
-			bar.append(' / ', getStrikeThrough(part));
+			breadcrumbs.push(getStrikeThrough(part));
 		} else {
-			const pathname = '/' + parts.slice(0, i + 1).join('/');
-			bar.append(i ? ' / ' : '', <a href={pathname}>{part}</a>);
```

Comment:
I had to refactor this because `i` was 0 but `0` was skipped, so the breadcrumbs would start with `/`, which we don't want